src/lib/y2storage/filesystems/type.rb

```diff
@@ -414,7 +414,11 @@ def boot_fstab_options(opt)
         case to_sym
         when :vfat
           # "iocharset=utf8" breaks VFAT case insensitivity (bsc#1080731)
-          opt.reject { |o| o == "iocharset=utf8" }
+          ret = opt.reject { |o| o == "iocharset=utf8" }
+          # Protecting vfat partitions in order to reduce security risks
+          # (bsc#1250510).
+          ret += ["dmask=0077"] unless ret.include?("dmask=0077")
```

Comment:
But what if the original options include something like `dmask=0447`. The idea is to avoid duplicated `dmask` options in general. Specially if the value is not the same.

In fact, if there is a contradictory value set by the user I'm not sure whether we should substitute it with our "0077" value.